### PR TITLE
Add vindex and 2048

### PR DESCRIPTION
**Pull request by wenxuanjun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,7 @@
 - [vcli](https://github.com/changhz/vcli) - A CLI tool to generate folder structure according to the [guideline](https://blog.vlang.io/the-complete-beginners-guide-to-cli-apps-in-v/)
 - [vgoogle](https://github.com/changhz/vgoogle) - Make google search on the terminal.
 - [vzcc](https://github.com/malisipi/vzcc) - A CLI cross-compiling tool based on Zig CC for V.
+- [vindex](https://github.com/wenxuanjun/vindex) - A simple file list server generating json strings, compatible with nginx's autoindex module.
```
Comment (JalonSolov) [inline README.md:82]: Please add items in alphabetical order.

File: README.md [hunk 2]
```diff
@@ -94,6 +95,7 @@
 - [Kurarin](https://github.com/FireRedz/kurarin) - osu! beatmap visualizer made in V. [Example video](https://p153.p0.n0.cdn.getcloudapp.com/items/6quvQjb5/ce3ea737-eb29-4b8c-a5f3-65a804a2f56f.mp4).
 - [minesweeper](https://github.com/ali-furkan/minesweeper-v) - A simple Minesweeper game written in vlang.
 - [v-pong](https://github.com/thebigsmileXD/v-pong) - A classic paddle game brought back to life through the power of V.
+- [2048](https://github.com/wenxuanjun/2048) - A 2048 game with several types of traditional AI integrated.
```
[the remark this answers is not in the document]
Comment (wenxuanjun) [reply README.md]: > Please add items in alphabetical order.

Rearranged, please take a look.

> Also, how is this different from the version included with V?

It contains completely rewritten performance optimized game logic, as well as six types of traditional AI, including an `expectimax` that can reach up to 120,000 points.

File: README.md (hunk 1 above)
Comment (JalonSolov) [reply README.md:82]: This entry should be between `vgoogle` and `vzcc`.

Comment (wenxuanjun) [reply README.md:81]: Sorry for not noticing that. It's now fixed and thanks a lot.

**Pull request merged**